cli/src/commands/edit.rs

```diff
@@ -21,10 +21,12 @@ use crate::cli_util::{CommandHelper, RevisionArg};
 use crate::command_error::CommandError;
 use crate::ui::Ui;
 
-/// Edit a commit in the working copy
+/// Loads the contents of the specified revision and sets it as the active changeset.
```

Comment:
nit: We use "commit" and "revision" quite interchangeably (maybe slightly preferring the latter?). Let's avoid also using "changeset".

Reply:
I noticed that, yeah, it's one of the reasons I opened #3249. I can replace changeset with revision.